### PR TITLE
[ADD]migrated attendance module in v11

### PR DESCRIPTION
**Pull request by Arpit-SerpentCS**

### CODE REVIEW

File: school_attendance/models/school_attendance.py [hunk 1]
```diff
@@ -1005,9 +1049,9 @@ class DailyAttendanceLine(models.Model):
     _order = 'roll_no'
     _rec_name = 'roll_no'
 
-    roll_no = fields.Integer('Roll No.', required=True, help='Roll Number')
+    roll_no = fields.Integer('Roll No.', help='Roll Number')
```
Comment (AnuKPatel) [inline school_attendance/models/school_attendance.py:1024]: @Arpit-SerpentCS Why you need to removed the constraint of required?

File: school_attendance/wizard/student_attendance_by_month_view.xml [hunk 2]
```diff
@@ -34,17 +34,18 @@
...
-        <record id="student_attendance_by_month_values" model="ir.values">
-            <field name="model_id" ref="school.model_student_student" />
-            <field name="object" eval="1" />
-            <field name="name">Student Attendances By Month</field>
-            <field name="key2">client_print_multi</field>
-            <field name="value" eval="'ir.actions.act_window,' + str(ref('action_student_attendance_by_month'))" />
-            <field name="key">action</field>
-            <field name="model">student.student</field>
-        </record>
+<!--         <record id="student_attendance_by_month_values" model="ir.ui.values"> -->
```
Comment (AnuKPatel) [inline school_attendance/wizard/student_attendance_by_month_view.xml]: @Arpit-SerpentCS  Remove the code which is commented or not useful.

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Remove `<!DOCTYPE html>` from all the xml file. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/21e29e4365536e3237f30fd32ed1da9b0b24d529/school_attendance/wizard/attendance_sheet_wizard_view.xml#L2

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Remove line no 5 and add "_" in 3rd line. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/21e29e4365536e3237f30fd32ed1da9b0b24d529/school_attendance/wizard/student_attendance_by_month.py#L5

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Use self.state in https://github.com/Arpit-SerpentCS/OdooEduERP/blob/21e29e4365536e3237f30fd32ed1da9b0b24d529/school_attendance/models/school_attendance.py#L139, L143, L147, L151

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Add domain in inside the search condition if it is used one time. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/1d1ffe1a9762d2ae1557a1059fbfdc8b2fff7a1f/school_attendance/models/school_attendance.py#L504, L505, L510

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Improve the README.rst file of a module. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/1d1ffe1a9762d2ae1557a1059fbfdc8b2fff7a1f/school_attendance/README.rst

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Please use DEFAULT_SERVER_DATE_FORMAT instead of `%Y-%m-%d` in everywhere in the code. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_library/school_attendance/models/school_attendance.py#L212, L224, L396, L403, L489, L505, L581

**Comment by Arpit-SerpentCS** [comment]
@murtuzasaleh Please review it. Made improvements and tested the flow according to comments.

cc @JayVora-SerpentCS

**Pull request merged**